docs/topics/rules-guide-intro.adoc

```diff
@@ -5,6 +5,6 @@
 
 This guide is for engineers, consultants, and others who want to create custom XML-based rules for {ProductName} ({ProductShortName}) tools.
 
-If you are new to {ProductShortName}, it is recommended that you start with the link:{ProductDocUserGuideURL}[_{GettingStartedBookName}_] for an overview of the {ProductName} features and system requirements. It is also recommended that you review the link:{ProductDocUserGuideURL}[_{UserCLIBookName}_], which provides detailed instructions on how to install and execute the {CLIName}.
+If you are new to {ProductShortName}, it is recommended that you start with the link:{ProductDocGettingStartedGuideURL}[_{GettingStartedBookName}_] for an overview of the {ProductName} features and system requirements. It is also recommended that you review the link:{ProductDocUserGuideURL}[_{UserCLIBookName}_], which provides detailed instructions on how to install and execute the {CLIName}.
```

Comment:
We don't say "It is recommended". Rewrite so that this does not appear. IBM

While you're at it, you should update the attribute name so that future TWs don't make the same mistake as you did -- not realizing that "Getting Started" = Introduction to.